src/ProviderHub/docs/Get-AzProviderHubProviderRegistration.md

```diff
@@ -40,6 +40,9 @@ Gets the provider registration details.
 PS C:\> Get-AzProviderHubProviderRegistration -ProviderNamespace "Microsoft.Contoso"
 ```
 
+Name Type
```

Comment:
It is better you could give an example with record

Reply:
For some reason it doesn't return the record. I do see the content in the response with the -Debug flag though.